controllers/record.go

```diff
@@ -31,8 +31,9 @@ import (
 // @Success 200 {object} object.Record The Response object
 // @router /get-records [get]
 func (c *ApiController) GetRecords() {
-	organization, ok := c.RequireAdmin()
-	if !ok {
+	user, ok := c.RequireSignedInUser()
+	if !ok || (!user.IsAdmin && !user.IsGlobalAdmin()) {
```

Comment:
can user be nil?
also move separate variable 
isNormalUser =  !(user.IsAdmin || user.IsGlobalAdmin())

Reply:
No it's cannot.
done